src/VSCodeExtension/package.json.v.template

```diff
@@ -1,6 +1,6 @@
 {
     "name": "quantum-devkit-vscode",
-    "version": "#VSCODEVSIX_VERSION#",
+    "version": "0.9999.22060152",
```

Comment:
Make sure that your v.template file keeps the #VSCODEVSIX_VERSION#. We can chat offline if you have questions about how this works.